contracts/external/DappHub/note.sol

```diff
@@ -34,8 +34,13 @@ contract DSNote {
             bar := calldataload(36)
         }
 
-        emit LogNote(msg.sig, msg.sender, foo, bar, msg.value, msg.data);
```

Comment:
This is a breaking change? Can't get msg.value directly?

Reply:
yea can't use msg.value directly, needs to be in an internal function

Reply:
`Using msg.value in non-payable functions (or introducing it via a modifier) is disallowed as a security feature.`